docs/index.md

```diff
@@ -0,0 +1,28 @@
+# TIDES
+
+Transit ITS Data Exchange Specification (TIDES) for historical transit operation data.
+
+This repository provides data schemas and tools to support the access, management, and improvement of historical transit operations data, including vehicle operations, passenger activity, fare collection, and other similar data.
+
+## TIDES specification
+
+The TIDES specification is maintained in the `/spec` sub-directory as a series of JSON tablesc ompatible with the [frictionless data](https://specs.frictionlessdata.io/table-schema/) table schema standards.
```

Comment:
typo: "tables ompatible"

Reply:
ooo - i actually meant to delete index.md and replace it with README.md...fixing now.